docs/make_docs.py

```diff
@@ -631,6 +639,13 @@ def run(self):
         default=False,
         help="Show Open3D version number rather than git hash.",
     )
+    parser.add_argument(
+        "--parallel",
+        action="store_true",
+        default=False,
```

Comment:
Do we want to make the default `True`?

Reply:
By default, we may not need the `--parallel` flag, as the error message will be hard to locate. The `--parallel` flag is only used when you know that the compilation will not have errors and you'll recompile the docs again and again.

Reply:
I added several examples instead

```bash
    # Run `python make_docs.py --help` to usage of the flags.
    python make_docs.py --help

    # Example: build .rst and C++ docs only, skip notebooks.
    python make_docs.py --execute_notebooks=never --sphinx --doxygen

    # Example: build .rst and C++ docs only, skip notebooks, with parallel build.
    python make_docs.py --execute_notebooks=never --sphinx --doxygen --parallel

    # Example: build .rst and c++ docs, execute notebooks when it has not been executed.
    python make_docs.py --execute_notebooks=auto --sphinx --doxygen
```